app/presenters/queries/content_item_presenter.rb

```diff
@@ -64,7 +65,16 @@ def latest
       end
 
       def join_supporting_objects(scope)
-        ChangeNote.join_editions(scope)
+        scope = ChangeNote.join_editions(scope)
+
+        if fields.include? :unpublishing
+          scope = scope.joins(
```

Comment:
For consistency with other models, it might be good to implement this as an `Unpublishing.join_editions` method.